openapi-spec/api.yaml

```
@@ -0,0 +1,811 @@
+      description: You do not have the required role to access this resource.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+          example:
+            timestamp: "2026-04-05T12:00:00Z"
+            status: 403
+            error: "Forbidden"
+            message: "Access denied: Teacher role required"
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)